TikTokApi/browser.py

```diff
@@ -120,6 +123,8 @@ def sign_url(self, **kwargs):
             for i in range(16)
         )
 
+        verifyFp = "verify_khgepmjd_L8THrD9Y_RqbR_4l9N_8cgF_dVPpzvJzxp9r"
```

Comment:
is this used somewhere?

Reply:
Oops, I left that in from testing just removed it.